front/src/config/i18n/fr.json

```diff
@@ -30,17 +30,17 @@
   },
   "calendar": {
     "allDay": "Toute la journée",
-    "previous": "précédent",
-    "next": "suivant",
-    "today": "aujourd'hui",
-    "month": "mois",
-    "week": "semaine",
-    "day": "jour",
+    "previous": "Précédent",
+    "next": "Suivant",
+    "today": "Aujourd'hui",
+    "month": "Mois",
+    "week": "Semaine",
+    "day": "Jour",
     "agenda": "Agenda",
-    "date": "date",
-    "time": "heure",
-    "event": "événement",
-    "noEventsInRange": "Aucuns évènements trouvés dans cette période.",
+    "date": "Date",
+    "time": "Heure",
+    "event": "Événement",
```

Comment:
Are you sure it makes sense to change those buttons to capitalize?

![Screenshot 2024-01-05 at 15 43 03](https://github.com/GladysAssistant/Gladys/assets/7365207/6eb5cfb6-b616-47c9-b4f9-6fc1b46fa522)

Reply:
They are capitalized in English and I think it looks cleaner, but that's just my opinion :)